src/mappings/mappingHandlers.ts

```diff
@@ -25,14 +17,15 @@ export function stripObjectUnicode(t: object): object {
 
 export async function handleBlock(block: NearBlock) {
   const txs = block.transactions.map((tx) => handleTransaction(tx));
-  const actions = block.actions.map((action) => handleAction(action));
+  const actions = block.actions.map(async (action) => await handleAction(action));
```

Comment:
This async stuff is getting funky. Maybe handle action could return an array of Actions instead then we can flatten and save as one